src/transformers/models/clip/modeling_clip.py

```diff
@@ -72,6 +72,76 @@ def clip_loss(similarity: torch.Tensor) -> torch.Tensor:
     return (caption_loss + image_loss) / 2.0
 
 
+@dataclass
+class CLIPVisionModelOutput(ModelOutput):
+    """
+    Base class for model's outputs that also contains a pooling of the last hidden states.
```

Comment:
```suggestion
    Base class for vision model's outputs that also contains a pooling of the last hidden states.
```

Reply:
Also didn't we create this class for the projection output? The pooling output is also part of the previous output "BaseModelOutputWithPooling" -> should we maybe focus on the `image_embeds` here instead of the `last_hidden_state`?

Reply:
Removed the `pooler_output`, but keeping the `last_hidden_state` as some users might still want it. But made `image_embeds` the first output.